packages/fxa-content-server/tests/server/amplitude.js

```diff
@@ -86,6 +88,88 @@ registerSuite('amplitude', {
       assert.doesNotThrow(() => amplitude(null, {}));
     },
 
+    'logs raw event with context': () => {
+      amplitudeConfig.rawEvents = true;
+      const event = {
+        type: 'oauth.signup.success',
+        offset: 3846,
+        time: 1585695795375,
+        flowTime: 3847,
+      };
+
+      const context = {
+        eventSource: 'content',
```

Comment:
Will this omit/prune empty or null values? I think it would be worthwhile to add mock values for all possible properties to verify.

Reply:
> Will this omit/prune empty or null values?

No, I think we'd want to do the least possible, not even pruning.

Reply:
I added more properties.

But also keep in mind that 

 - the `data` arg in the amplitude calls varies, so it's most likely not as large as this test lead you to believe
 - lodash's `pick` will omit properties that do not exist on the source object